cluster/images/conformance/run_e2e.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
should this script have `errexit`, `nounset`, and `pipefail` per usual?

Reply:
fixing